src/client/method-status.c

```diff
@@ -25,6 +27,25 @@ struct bus_properties_map {
         size_t offset;
 };
 
+typedef struct method_status_change method_status_change;
+
+struct method_status_change {
+        Client *client;
+        char **units;
+        char *node_name;
+        size_t units_count;
+        size_t max_len;
+};
+
+void free_method_status_change(method_status_change *s) {
+        free(s);
+}
+
+void clear_screen() {
+        printf("\033[2J");
+        printf("\033[%d;%dH", 0, 0);
```

Comment:
Are we sure that this works on all terminals? Or at least those standard one?

Reply:
These are [ANSI escape codes](https://en.wikipedia.org/wiki/ANSI_escape_code). They should be supported pretty much everywhere.

Reply:
OK